src/DependencyInjection/Compiler/MetadataEnricherPass.php

```diff
@@ -41,7 +43,8 @@ public function process(ContainerBuilder $container)
 
             $metadataEnricherId = sprintf('prooph_event_store.%s.%s', 'metadata_enricher_plugin', $name);
             $metadataEnricherDefinition = $container->getDefinition($metadataEnricherId);
-            $metadataEnricherDefinition->setClass('%prooph_event_store.metadata_enricher_plugin.class%');
+            $metadataEnricherDefinition->setClass(MetadataEnricherPlugin::class);
+            $metadataEnricherDefinition->addTag(sprintf('prooph_event_store.%s.plugin', $name));
```

Comment:
Move this to `ProophEventStoreExtension`, it will fix the enrichers (`MetadataEnricherPass` is executed after the `PluginPass`).